src/Build/Evaluation/LazyItemEvaluator.cs

```diff
@@ -87,6 +88,7 @@ public bool EvaluateConditionWithCurrentState(ProjectElement element, ExpanderOp
             {
                 return true;
             }
+            EvaluateConditionEventSource.Log.EvaluateConditionStart(String.Format("Evaluate Condition {0} - Begin", condition));
```

Comment:
Is it worth the allocations and time to do a `string.Format` here, as opposed to just conveying the "Evaluate Condition - Begin" information with the event type?

Reply:
(general question, not just for this specific instance of the pattern)